.github/workflows/docker.yml

```diff
@@ -0,0 +1,53 @@
+name: build docker images
+
+on:
+  push:
+
+jobs:
+  build-check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check Out Repo
+        uses: actions/checkout@v2
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Build and push
+        id: docker_build
+        uses: docker/build-push-action@v2
+        with:
+          context: ./components/check
+          file: ./components/check/Dockerfile
+          push: false
```

Comment:
and we could try to push to gitlab container repository or even official docker repository since the code is public and based on open-source